presentation/src/main/java/com/android254/presentation/home/viewmodel/HomeViewModel.kt

```diff
@@ -41,7 +41,11 @@ class HomeViewModel @Inject constructor(
     var viewState by mutableStateOf(HomeViewState())
         private set
 
-    fun onGetHomeScreenDetails() {
+    init {
+        onGetHomeScreenDetails()
```

Comment:
@yveskalume does this affect tests in any way?

Reply:
No, there isn't any test for HomeViewModel